Data/Matrix.hs

```diff
@@ -339,7 +339,9 @@ fromList :: Int -- ^ Rows
          -> [a] -- ^ List of elements
          -> Matrix a
 {-# INLINE fromList #-}
-fromList n m = M n m 0 0 m . V.fromListN (n*m)
+fromList n m xs
+    | n*m > (1+length xs) = error $ "fromList: The list argument must have at least rows*cols elements. (" ++ (sizeStr n m) ++ ")"
```

Comment:
That's not correct. I mistakenly thought it had to be there because, as mentioned in #64, the fromList function only doesn't work if `rows*cols > (1+length list)`. But obviously the case where `(length list) = (rows*cols)-1)` is covered by `rows*cols > length list` as well. I will modify the commit.